docs/PokemonCreation.md

```diff
@@ -0,0 +1,68 @@
+## Pokemon Creation
+
+Pokemon creation in FullScreenPokemon consists of three different types, Unit Test Pokemon creation, Trainer Battle Pokemon creation, and Wild Pokemon Grass Encounter creation.
```

Comment:
I don't think the three main types should be listed here. It's very likely other types will be added in the future. 

Instead, mention them as they come later on.